apps/ui-kit/src/lib/styles/index.css

```diff
@@ -45,6 +45,18 @@
     .text-label-md {
         font-variant-alternates: 'ss01';
     }
+
+    input:-webkit-autofill,
+    input:-webkit-autofill:enabled,
+    input:-webkit-autofill:hover,
+    input:-webkit-autofill:focus {
+        appearance: none;
+        background-image: none !important;
+        background-color: transparent !important;
+        color: #545e6e !important;
+        transition: background-color 5000s ease-in-out 0s;
```

Comment:
Why is the transition needed?

Reply:
5s 💀

Reply:
the background-color transition prevents a flicker when browsers like Chrome apply autofill's default color (yellow or gray) before custom styles take effect and the long duration (5000s) ensures the change is imperceptible